test/decoder.js

```diff
@@ -60,6 +60,23 @@ test('decode wiki example', function(done) {
 
     decoder.write(msg.replace(/\|/g, sep));
 });
+test('Filled up buffer', function(done) {
```

Comment:
lets add a comment saying what this was testing or reference the PR that way we remember what the purpose here is